communication.rst

```diff
@@ -73,24 +89,80 @@ RSS feed readers.
 .. _tutor: https://mail.python.org/mailman/listinfo/tutor
 .. _StackOverflow: https://stackoverflow.com/
 .. _Libera.Chat: https://libera.chat/
+.. _web gateway: https://mail.python.org/archives/
 
 
-Discourse
----------
+Discourse (discuss.python.org web forum)
+----------------------------------------
 
 We have our own `Discourse`_ forum for both developers and users. This forum
 complements the `python-dev`_, `python-ideas`_, `python-help`_, and
-`python-list`_ mailing lists. Also, voting for new core developers takes place
-at `Discourse`_.
+`python-list`_ mailing lists.
+
+Most core development discussions take place in the open forum categories for
+`PEPs`_ and `Core Development`_. (These are the Discourse equivalents to the
+python-dev mailing list)
+
+The `Committers`_ category restricts posting to core developers only, and is
+used more for announcements and notifications, rather than for discussions. It
+is also the venue for core developer promotion votes. (This category is the
+equivalent of the python-committers mailing list)
+
+There is a final, rarely used, core development category on Discourse that is
+only visible to core developers. This can be used to share administrative
+information with all core developers in a non-public forum (e.g. logistics for
+in person core development sprints), as well as for individual core developers
+to share info that they'd like other core devs to be aware of, but would prefer
+not to have permanently archived on the internet.
 
 .. _Discourse: https://discuss.python.org/
+.. _PEPs: https://discuss.python.org/c/peps/
+.. _Core Development: https://discuss.python.org/c/core-dev/
+.. _Committers: https://discuss.python.org/c/committers/
+
+
+Discord (private chat server)
+-----------------------------
+
+For more real-time discussions, the core development team have a private Discord
+server available. Core developers, Steering Council members, triagers, and
+documentarians on the project are eligible to join the server. Joining the
+Discord server is entirely optional, as all essential communications occur on
+the mailing lists and Discourse forums.
+
+For core developers, a long lived multiple use invitation link for this server
+can be found in the private core developer only section of the Discourse forum.
+
+For triagers and documentarians joining the Discord server, a single use invitation
+link should be generated and sent to them directly.
+
+When first joining the server, new users will only have access to the ``#welcome``
+and ``#rules-and-info`` channels. To link their Discord ID with their project
+role, core developers may update their Steering Council `voter record`_ with
```

Comment:
This repo is private but returns a 404 if you have no access, which looks like a broken link.

I suggest putting a lock icon next to it, as has been done here: https://devguide.python.org/coredev/#gaining-commit-privileges

```suggestion
role, core developers may update their Steering Council 🔒 `voter record`_ with
```